.example.env

```diff
@@ -8,11 +8,10 @@ GADS_CUSTOMER_CLIENT_LINK=<YOUR_GADS_CUSTOMER_CLIENT_LINK>
 
 GADS_REFRESH_TOKEN_WITH_METRICS=<YOUR_GADS_REFRESH_TOKEN_WITH_METRICS>
 GADS_CID_WITH_METRICS=<YOUR_GADS_CID_WITH_METRICS>
-GADS_LOGIN_CUSTOMER_ID=<YOUR_GADS_LOGIN_CUSTOMER_ID>
```

Comment:
Good spot here!

I believe this is actually a mistake on our part naming one of the examples incorrectly, this should be:

```env
GADS_LOGIN_CUSTOMER_ID_WITH_METRICS=<YOUR_GADS_LOGIN_CUSTOMER_ID_WITH_METRICS>
```

If you could simply replace the duplicate with above, would be appreciated

Reply:
Added GADS_LOGIN_CUSTOMER_ID_WITH_METRICS